game-core/src/main/java/games/strategy/triplea/delegate/battle/grouptarget/FiringGroup.java

```diff
@@ -0,0 +1,50 @@
+package games.strategy.triplea.delegate.battle.grouptarget;
+
+import games.strategy.engine.data.Unit;
+import games.strategy.engine.data.UnitType;
+import games.strategy.triplea.delegate.Matches;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import lombok.Value;
+import org.triplea.java.collections.CollectionUtils;
+
+@Value(staticConstructor = "of")
+public class FiringGroup {
+  Collection<Unit> firingGroup;
```

Comment:
Should these be private & final?

Reply:
@Value makes them private & final.

Reply:
Ah, same thing! Very confusing when you're not familiar with it.

Reply:
IDEA even generates unnecessary 'private & final' modifiers for `@Value`. IMO this is one where it's okay to leave off, some familiarity with lombok is perhaps required. Making methods explicit on `@UtilityClass` IMO is okay (more helpful as you might not always often see the annotation on the same screen as the method)